app/controllers/qualifications/errors_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module Qualifications
+  class ErrorsController < QualificationsInterfaceController
+    skip_before_action :verify_authenticity_token
+    skip_before_action :authenticate_user!
+    skip_before_action :handle_expired_token!
+
+    def not_found
```

Comment:
Is it worth adding these identical controller hooks/actions to a common concern?